iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/DanglingLineScalable.java

```diff
@@ -109,7 +109,7 @@ public void filterInjections(Network n, List<Injection> injections, List<String>
      * If scalingConvention is GENERATOR, the load active power decreases for positive "asked" and increases inversely
      */
     @Override
-    public double scale(Network n, double asked, Scalable.ScalingConvention scalingConvention) {
+    public double scale(Network n, double asked, Scalable.ScalingConvention scalingConvention, boolean reconnect) {
```

Comment:
We really should consider grouping all theses parameters in a ScalingContext object.